mono/utils/mono-threads.h

```diff
@@ -532,6 +532,13 @@ mono_native_thread_create (MonoNativeThreadId *tid, gpointer func, gpointer arg)
 MONO_API void
 mono_native_thread_set_name (MonoNativeThreadId tid, const char *name);
 
+#ifdef HOST_WIN32
+void
+mono_native_thread_set_namew (HANDLE threadHandle, const wchar_t *name);
+#else
+#define mono_native_thread_set_namew(handle, name) /* nothing */
```

Comment:
Macro helps, to avoid evaluating native_handle.

Reply:
Trailing "W" in Windows means wide/wchar_t/unicode. Perhaps Mono would say _utf16, perhaps w is ok and clear. In this case it could also mean Windows.

Reply:
I would vote for _utf16 as it's explicit to non-Windows devs and consistent with rest of mono.